grafanalib/core.py

```diff
@@ -381,8 +385,9 @@ class Row(object):
         default=True, validator=instance_of(bool),
     )
     height = attr.ib(default=DEFAULT_ROW_HEIGHT, validator=instance_of(Pixels))
-    showTitle = attr.ib(default=None)
+    showTitle = attr.ib(default=True)
```

Comment:
Why are you changing this default?

Reply:
Probably by mistake, gonna guess I thought I added it myself and that it gave a different result than I thought it should while porting these JSON dashboards.